0000-template.md

```diff
@@ -4,8 +4,9 @@
 * **Tracking Issue**: #{TRACKING_ISSUE}
 * **API Bar Raiser**: @{BAR_RAISER_USER}
 
-> Write one sentence which is a brief description of the feature from a user
-> perspective ("impact on users").
+> Write one sentence which is a brief description of the feature. It should describe:
```

Comment:
I still think it will be easier to review the RFCs with a separation for two sections. I think the customer pain is important to have its own section. But let's see what other people think. Maybe it's just me :)

Reply:
Maybe we could add some redundancy to adding this to line 75 as well:

```
### Why are we doing this?

> What is the motivation for this change?
+ > What is the customer pain this addresses?
```

Reply:
I like that. Let's add there as well